### ISSUE TITLE
Quantity available not available for the default product sku?

### ISSUE THREAD

**Issue by droyop** [BroadleafCommerce/BroadleafCommerce#1826]
Hi,

We migrated from 4.x to 5.3 and the admin UI doesn't show a quantity available field for the products. 

We've observed the field is available if we create a separate sku for the product, but not for the implicit sku.

Is it not expected to control the inventory via the default SKUs anymore?

We checked and the functionality is intact (After a purchase, the default sku quantity available is decreased, and the presentation layer gets the quantity available just fine), it's just the admin which doesn't show it.

**Comment by droyop** [comment]
@phillipuniverse is there anything we are doing wrong here? We can maintain the stock for the products directly in the default sku in the database, but we cannot set the quantity in the admin

**Comment by phillipuniverse** [comment]
Sorry for the late reply here, the issue was when we first created the community version, we kept around some of the overrides necessary for our enterprise edition. In the case of quantities, we have something else that manages inventory across different locations, and so managing the single property on the Sku doesn't make sense.

I have fixed this in BroadleafCommerce/DemoSite@2d8b67dee5e4d82fd192d61fed04c50357ef30aa. You will need to make a similar change in your existing project.
